<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1210</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(27, 39, 50, 255),stop:1 rgba(47, 53, 74, 255));
	color: #000000;
background-repeat: no-repeat; 
background-position: center;}</string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1210</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(27, 39, 50, 255),stop:1 rgba(47, 53, 74, 255));
	color: #000000;
background-repeat: no-repeat; 
background-position: center;}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>611</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 39pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string>Search Car Information</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>180</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:20px;
color: rgb(0, 0, 0);
background-color: rgb(0, 255, 203);
font: 75 25pt &quot;URW Bookman L&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="pl2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>190</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Ubuntu Mono&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>180</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string>Plate Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>530</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>370</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string>Plate Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>600</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string>Make</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>450</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
    <widget class="QLabel" name="pldb">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>370</y>
       <width>671</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="colordb">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>530</y>
       <width>671</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="namedb">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>290</y>
       <width>671</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="makedb">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>600</y>
       <width>671</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="modeldb">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>450</y>
       <width>671</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="back4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:20px;
color: rgb(0, 0, 0);
background-color: rgb(0, 255, 203);
font: 75 25pt &quot;URW Bookman L&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="error">
     <property name="geometry">
      <rect>
       <x>246</x>
       <y>650</y>
       <width>451</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;URW Bookman L&quot;;
color:rgb(239, 41, 41)
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
